e2e/e2etests/test_migrate_tss.go

```diff
@@ -0,0 +1,182 @@
+package e2etests
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strconv"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/btcsuite/btcutil"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/zetacore/e2e/runner"
+	"github.com/zeta-chain/zetacore/e2e/utils"
+	zetacrypto "github.com/zeta-chain/zetacore/pkg/crypto"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func TestMigrateTss(r *runner.E2ERunner, _ []string) {
+	r.SetBtcAddress(r.Name, false)
+	stop := r.MineBlocksIfLocalBitcoin()
+	defer stop()
+
+	// Pause inbound procoessing for tss migration
+	r.Logger.Info("Pause inbound  processing")
+	msg := observertypes.NewMsgDisableCCTX(
+		r.ZetaTxServer.GetAccountAddress(0),
+		false,
+		true)
+	_, err := r.ZetaTxServer.BroadcastTx(utils.FungibleAdminName, msg)
+	require.NoError(r, err)
+
+	// Migrate btc
+	// Fetch balance of BTC TSS address
+	utxos, err := r.GetTop20UTXOsForTssAddress()
+	require.NoError(r, err)
+
+	var btcBalance float64
+	for _, utxo := range utxos {
+		btcBalance += utxo.Amount
+	}
+
+	// Use fixed fee for migration
+	btcTSSBalanceOld := btcBalance
+	fees := 0.01
```

Comment:
Why defining a variable if only used once? We should rather define a const in this case